src/Hooks/useCheckFriendStatus.tsx

```diff
@@ -0,0 +1,15 @@
+import { useState,useEffect } from "react";
+
+export const useCheckFriendStatus=(isOnline:boolean)=>{
```

Comment:
Please follow proper indentation. Have you not setup prettier for that as we discussed earlier ?